### ISSUE TITLE
Memory Allocation Error When Reading a .C3D

### ISSUE THREAD

**Issue by henrypowell87** [pyomeca/ezc3d#109]
Hi there,

I'm working on a python script to format the CMU Mocap dataset ([http://mocap.cs.cmu.edu/faqs.php](url)) from .c3d files to .csv files of [frames * makers] dimensional arrays for each captured movement. For the most part simply importing each .c3d file and formatting the data to the correct shape and type works fine. However, in some cases the script throws a memory allocation error:

```console
python3(1329,0x108bf65c0) malloc: *** error for object 0x7feefb0d7588: pointer being freed was not allocated
python3(1329,0x108bf65c0) malloc: *** set a breakpoint in malloc_error_break to debug
Abort trap: 6
```

After attempting some debugging I noticed the error is thrown in two at least two instances:
I. On a small number of .c3d files from the data that exist in the given directory,
II. When the input string to the ```c3d()``` function points to a non existent file.

Here's two minimal working examples that throw the error on my machine:

```python
from ezc3d import c3d

c3d_path = './DataSets/CMU/subjects/06/06_02.c3d'
c = c3d(c3d_path)
```

In the above case, the file 06_02.c3d exists and is able to be parsed by another python libary I tested (c3d.py).

```python
from ezc3d import c3d

c3d_path = './DataSets/CMU/subjects/06/nlasjkasdajklslkj.c3d'
c = c3d(c3d_path)
```

In the above case, the file does not exist and the same error is thrown.

I'm working on:
OSX Mojave 10.14.6
Python 3.7.4

Exc3d version was that installed through conda using ```conda install -c conda-forge ezc3d``` only a couple of days ago.

Any help with this would be great and please let me know if you need any more info.

Cheers,

Henry

**Comment by pariterre** [comment]
Dear @henrypowell87 

Thank you for reporting this bug! It seems that the bug you describe is Mac related. I ran your minimal program and could not reproduce it from my Ubuntu 18.10. 
I do not have access to a Mac for the weekend. I'll ask a colleague at the lab on Monday to see if I can reproduce the bug!

I let you know ASAP :)

**Comment by pariterre** [comment]
@henrypowell87 
Sorry for the delay! I've had a lot of things to do during the last weeks. 

I tried to reproduce the error you reported using the `06_02.c3d` but I could not reproduce it... Using both Linux and MacOSX version installed the last version on conda, I was able to read the file properly. 
I must admit that I am a little clueless about the problem then... Maybe the path to the file was not right?

Did you manage to read the file?
